sunpy/data/data_manager/tests/test_manager.py

```diff
@@ -144,3 +145,26 @@ def test_file_changed(data_function, storage):
     # Now it should error
     with pytest.warns(SunpyUserWarning):
         data_function()
+
+
+def test_delete_db(manager, sqlstorage):
```

Comment:
Have you ran this test without the fix you added to make sure it fails?

Reply:
I had not, and turns out it passed. I've updated the test now - it fails without the bug fix, and passes with it.